Tasks/Maven/mavenHelper.ps1

```diff
@@ -0,0 +1,177 @@
+function ConfigureJDK
```

Comment:
This code has previously been reviewed by folks in the Java ALM team. I decided to create a new pull request because too many changes have happened in the meantime (we were blocked from completing this task because decoding the generic endpoint in js).